README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
I added this because I was encountering errors in multiple of the steps below until I had both of these running. I don't remember exactly which steps required which processes precisely, @orangejenny or @esoergel might. But definitely couldn't get through these steps successfully without them.